ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -160,7 +160,30 @@ class _ConnectionKey(NamedTuple):
     instrument_type_id: str
 
 
-class BaseReservation(abc.ABC):
+class _BaseSessionContainer(abc.ABC):
+    """Manages Session Management Client."""
+
+    def __init__(
+        self,
+        session_manager: SessionManagementClient,
+    ) -> None:
+        """Initialize session manager object."""
```

Comment:
Please do not list out all the implementation details in the docstring.

```suggestion
        """Initialize the base session container."""
```